courses/models.py

```diff
@@ -125,6 +126,12 @@ def duration(self):
         """Gets the duration from the associated Page if it exists"""
         return self.page.duration if self.page else None
 
+    @property
+    def outcomes(self):
+        """Gets the learning outcomes from the associated Page children if it exists"""
+        children = self.page.get_children()
+        return children.type(LearningOutComesPage).first().specific
```

Comment:
How about `children.type(LearningOutcomesPage).first().specific if self.page and self.page.children.type(LearningOutcomesPage).count() else None`